pkg/utils/dataset.go

```diff
@@ -73,7 +73,8 @@ func GetAccessModesOfDataset(client client.Client, name, namespace string) (acce
 func IsTargetPathUnderFluidNativeMounts(targetPath string, dataset datav1alpha1.Dataset) bool {
 	for _, mount := range dataset.Spec.Mounts {
 
-		mPath := UFSPathBuilder{}.GenAlluxioMountPath(mount, dataset.Spec.Mounts)
+		mAlluxioPath := UFSPathBuilder{}.GenAlluxioMountPath(mount, dataset.Spec.Mounts)
+		mGooseFSPath := UFSPathBuilder{}.GenGooseFSMountPath(mount, dataset.Spec.Mounts)
```

Comment:
I'm wonder what does the `mGooseFSPath` look like? Do we need duplicate `GenGooseFSUFSRootPath`  ? What's your opinion? @allenhaozi

Reply:
Can a DataSet be used by two Runtime simultaneously ? 
Or whether GooseFS can follow the same rules as Aluxio,  

So we don't need a duplicate `GenXXXMountPath`

Reply:
Firstly, Dataset should can be used for different type runtime. 
Secondly, at present, GooseFS can follow ths same rules as Alluxio, but the future may be different.